src/SFA.DAS.AANHub.Application/Members/Commands/PatchMember/PatchMemberCommandHandler.cs

```diff
@@ -47,8 +53,29 @@ public async Task<ValidatedResponse<SuccessCommandResult>> Handle(PatchMemberCom
 
         _auditWriteRepository.Create(audit);
 
-        await _aanDataContext.SaveChangesAsync(cancellationToken);
+        if (member.Status.ToLower() == "withdrawn") { CreateNotification(member); }
```

Comment:
make the "withdrawn" a constant rather than a magic string.   Then change any usage of "withdrawn" in the tests to use that constant too.